src/service/main.cpp

```diff
@@ -158,8 +164,17 @@ int main(int argc, char **argv) {
     const opencmw::zmq::Context                               zctx{};
     std::vector<std::unique_ptr<opencmw::client::ClientBase>> clients;
     clients.emplace_back(std::make_unique<opencmw::client::MDClientCtx>(zctx, 20ms, ""));
+    clients.emplace_back(std::make_unique<opencmw::client::RestClient>(opencmw::client::DefaultContentTypeHeader(opencmw::MIME::BINARY)));
     opencmw::client::ClientContext client{ std::move(clients) };
 
+    // create example signals
+    opencmw::service::dns::DnsClient dns_client{client, "http://localhost:8080/dns"};
+    dns_client.registerSignals({
+        {"http", "localhost", 8080, "service 1", "service"},
+        {"http", "localhost", 8080, "service 2", "service"},
+        {"http", "localhost", 8080, "service 3", "service"},
+        {"http", "localhost", 8080, "service 4", "service"}
```

Comment:
small nit: having more realistic/diverse sample signals here would be nicer, the only thing we can really filter on is the signal name now.